### PR TITLE
Fix/integer conversion using primitive operator library

### PR DESCRIPTION
**Pull request by Nizal-Shammry**
Replace Clojure-based arithmetic and bitwise with the new Integer primitive-operator library

### CODE REVIEW

File: src/identitas/proquint.clj [hunk 1]
```diff
@@ -15,11 +15,11 @@ equivalent."
 (def ^:private int-to-vowel
   '[\a \i \o \u])
 
-(def ^:private mask-4 0xF0000000)
-(def ^:private mask-2 0xC0000000)
+(def ^:private mask-4 (unchecked-int 0xF0000000))
```
Comment (phillord) [inline src/identitas/proquint.clj:18]: This should be `int` not `unchecked-int`. Doesn't make any practical difference but clearer.

File: project.clj [hunk 2]
```diff
@@ -2,4 +2,4 @@
   :description "Mechanisms for generating and manipulating identifiers."
   ;;:url "http://example.com/FIXME"
   :dependencies [[org.clojure/clojure "1.7.0"]
-                 [primitive-operator "0.2-SNAPSHOT"]])
+                 [primitive-operator "0.1-SNAPSHOT"]])
```
Comment (phillord) [inline project.clj]: `0.1` not `0.1-SNAPSHOT`

File: src/identitas/proquint.clj (hunk 1 above)
Comment (Nizal-Shammry) [reply src/identitas/proquint.clj:18]: I put as (def ^:private mask-4 (int 0xF0000000)) but it thorw Error compiling

Comment (Nizal-Shammry) [inline src/identitas/proquint.clj]: Unhandled clojure.lang.Compiler$CompilerException
   Error compiling proquint.clj at (18:25)

             Compiler.java: 1699  clojure.lang.Compiler$StaticMethodExpr/eval
             Compiler.java:  439  clojure.lang.Compiler$DefExpr/eval
             Compiler.java: 6787  clojure.lang.Compiler/eval
             Compiler.java: 7227  clojure.lang.Compiler/load
                      REPL:    1  user/eval10296
             Compiler.java: 6782  clojure.lang.Compiler/eval
             Compiler.java: 6745  clojure.lang.Compiler/eval
                  core.clj: 3081  clojure.core/eval
                  main.clj:  240  clojure.main/repl/read-eval-print/fn
                  main.clj:  240  clojure.main/repl/read-eval-print
                  main.clj:  258  clojure.main/repl/fn
                  main.clj:  258  clojure.main/repl
               RestFn.java: 1523  clojure.lang.RestFn/invoke
    interruptible_eval.clj:   87  clojure.tools.nrepl.middleware.interruptible-eval/evaluate/fn
                  AFn.java:  152  clojure.lang.AFn/applyToHelper
                  AFn.java:  144  clojure.lang.AFn/applyTo
                  core.clj:  630  clojure.core/apply
                  core.clj: 1868  clojure.core/with-bindings*
               RestFn.java:  425  clojure.lang.RestFn/invoke
    interruptible_eval.clj:   85  clojure.tools.nrepl.middleware.interruptible-eval/evaluate
    interruptible_eval.clj:  222  clojure.tools.nrepl.middleware.interruptible-eval/interruptible-eval/fn/fn
    interruptible_eval.clj:  190  clojure.tools.nrepl.middleware.interruptible-eval/run-next/fn
                  AFn.java:   22  clojure.lang.AFn/run
   ThreadPoolExecutor.java: 1142  java.util.concurrent.ThreadPoolExecutor/runWorker
   ThreadPoolExecutor.java:  617  java.util.concurrent.ThreadPoolExecutor$Worker/run
               Thread.java:  748  java.lang.Thread/run

1. Caused by java.lang.IllegalArgumentException
   Value out of range for int: 4026531840

Comment (Nizal-Shammry) [reply src/identitas/proquint.clj:18]: Caused by java.lang.IllegalArgumentException
   Value out of range for int: 4026531840

Comment (phillord) [reply src/identitas/proquint.clj:18]: Yes, sorry you are correct. Just ignore

**Pull request closed without merging**